packages/main/src/themes/TreeListItem.css

```diff
@@ -89,24 +89,24 @@
     color: var(--sapList_Active_TextColor);
 }
 
-/* [ui5-li]: infoState */
-:host([info-state="Warning"]) .ui5-li-tree-info {
+/* [ui5-li]: additionalTextState */
+:host([additional-text-state="Warning"]) .ui5-li-tree-info {
```

Comment:
.ui5-li-tree-info class should be also renamed perhaps, but check if it is going to be removed by this PR https://github.com/SAP/ui5-webcomponents/pull/3280